i18n/zh-cn/docusaurus-plugin-content-docs/current/dev/mode/at-mode.md

```diff
@@ -193,7 +193,7 @@ UNDO_LOG Table：不同数据库在类型上会略有差别。
 | log_status    | tinyint      |
 | log_created   | datetime     |
 | log_modified  | datetime     |
-
+> 请确保branche_id中不包含逗号
```

Comment:
Why branch_id as bigint type may contain a comma ?

Reply:
sorry,i made a mistake and i will correct it.Thank you for your review.